simulator/heap.h

```diff
@@ -54,7 +52,7 @@ class HeapObject
       parent(0),
       rank(0),
       pointsTo(0),
-      size(1),
```

Comment:
Perhaps there is a reason that the size starts from 1? We don't have an integration test yet, right? I'd not make this semantic change without an integration test.

Reply:
No integration test yet. I think it might be a little hard since trace events have changed between the two versions so there is not backward compatibility.

Reply:
I restored the bytes variable, and initialize size to 1.